test/bitgo/keyutil.js

```diff
@@ -0,0 +1,73 @@
+/* global describe, it */
+
+const assert = require('assert')
+var randombytes = require('randombytes')
+
+const BigInteger = require('bigi')
+
+const {
+  bitgo: {
+    keyutil: {
+      privateKeyBufferToECPair,
+      privateKeyBufferFromECPair
+    }
+  },
+  ECPair
+} = require('../../src')
+
+describe('privateKeyBufferFromECPair', function () {
+  it('pads short private keys', function () {
+    var keyPair = new ECPair(BigInteger.ONE)
+    assert.strictEqual(privateKeyBufferFromECPair(keyPair).byteLength, 32)
+    assert.strictEqual(privateKeyBufferFromECPair(keyPair).toString('hex'),
+      '0000000000000000000000000000000000000000000000000000000000000001')
+  })
+
+  it('does not pad 32 bytes private keys', function () {
+    var hexString = 'a000000000000000000000000000000000000000000000000000000000000000'
+    var keyPair = new ECPair(new BigInteger(hexString, 16))
+    assert.strictEqual(privateKeyBufferFromECPair(keyPair).byteLength, 32)
+    assert.strictEqual(privateKeyBufferFromECPair(keyPair).toString('hex'), hexString)
+  })
+
+  it('throws if the key is too long', function () {
+    var hexString = '10000000000000000000000000000000000000000000000000000000000000000'
+
+    assert.throws(function () {
+      var keyPair = new ECPair(new BigInteger(hexString, 16))
+      privateKeyBufferFromECPair(keyPair)
+    }, new RegExp('Private key must be less than the curve order'))
+  })
+})
+
+describe('privateKeyBufferToECPair', function () {
+  it('constructs an ECPair from a random private key buffer', function () {
+    var prvKeyBuffer = randombytes(32)
+    var ecPair = privateKeyBufferToECPair(prvKeyBuffer)
+    var ecPairPrvBuffer = privateKeyBufferFromECPair(ecPair)
+    assert.strictEqual(Buffer.compare(ecPairPrvBuffer, prvKeyBuffer), 0)
+  })
+
+  it('throws if the private key is out of range', function () {
+    var prvKeyBuffer = Buffer.alloc(32, 0xff)
```

Comment:
Perhaps add a source here such as for example https://en.bitcoin.it/wiki/Private_key#Range_of_valid_ECDSA_private_keys

Reply:
ok

Reply:
actually the test is useless since the range is checked by `ECPair` and not our func. The helper funcs actually just do BigInteger <> Buffer conversion.

Reply:
Please remove any useless tests, or if they're testing something else, put them in an appropriate different place.